python/rascaline/tests/utils/correlate_density.py

```diff
@@ -208,19 +196,7 @@ def test_so3_equivariance(frames, nu_target, angular_cutoff, selected_keys):
 )
 @pytest.mark.parametrize(
     "frames, nu_target, angular_cutoff, selected_keys",
-    [
-        (
-            h2_isolated(),
-            3,
-            None,
-            Labels(
-                names=["spherical_harmonics_l"],
-                values=np.array([0, 4, 5]).reshape(-1, 1),
-            ),
-        ),
-        (h2o_isolated(), 2, 5, None),
-        (h2o_periodic(), 2, 5, None),
-    ],
+    [(h2_isolated(), 2, 3, None)],
 )
 def test_o3_equivariance(frames, nu_target, angular_cutoff, selected_keys):
```

Comment:
same comment, if we only have one parametrization and are not using fixtures, we should create the variables inside the function directly